src/meta/src/hummock/compactor_manager.rs

```diff
@@ -321,15 +332,19 @@ impl CompactorManagerInner {
         let mut cancel_tasks = vec![];
         for progress in progress_list {
             if let Some(task_ref) = self.task_heartbeats.get_mut(&progress.task_id) {
+                task_ref.update_at = now;
```

Comment:
In original implement, we update "expire_at" to "expire_at + 60 secs". And we check it with "now - 30secs". It means that the task will be cancled after "90s" not changed.
But now, we will cancel task if it did not receive heartbeat during "30s".  Is it too short ?